src/components/KeyAddedModal.tsx

```diff
@@ -14,12 +14,16 @@ const KeyAddedModal = ({ visible, close, signer }) => {
 
   const signerTypeConfig = {
     [SignerType.KEEPER]: {
```

Comment:
It now makes it the same as defaultConfig, so why not just remove the special config here so it will use the defaultConfig and avoid the duplication?

Reply:
Removes extra configuration. Making the modal consistent for External key and other modals.
Removes text "You can associate a contact with this key if you wish to"
![EK](https://github.com/user-attachments/assets/0c8f4af4-abe2-41fb-9a30-b4b15006281f)